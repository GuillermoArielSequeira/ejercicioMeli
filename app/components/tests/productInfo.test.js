const React = require('react');
const { shallow } = require('enzyme');
const ProductInfo = require('../productInfo');

describe('productInfo Component', () => {
  const props = {
    title: 'title product',
    picture: 'https://http2.mlstatic.com/D_705684-MLA44922791090_022021-O.jpg',
    price: { currency: 'ARS', amount: '62,975', decimals: 0 },
    condition: 'new',
    sold_quantity: 5,
    description: 'una descripciom',
    breadcrumb: [
      { id: 'MLA1648', name: 'ComputaciÃ³n' },
      { id: 'MLA430687', name: 'Laptops y Accesorios' },
      { id: 'MLA1652', name: 'Notebooks' }
    ]
  }

  it('render productInfo correctly', () => {
    const wrapper = shallow(<ProductInfo {...props} />);
    expect(wrapper).toMatchSnapshot();
  })
});
