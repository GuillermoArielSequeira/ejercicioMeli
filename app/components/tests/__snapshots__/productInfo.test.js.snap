// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`productInfo Component render productInfo correctly 1`] = `ShallowWrapper {}`;
